lib/api/CDataFrameAnalysisInstrumentation.cc

```diff
@@ -190,9 +190,11 @@ void CDataFrameAnalysisInstrumentation::monitor(CDataFrameAnalysisInstrumentatio
             instrumentation.memoryReestimate(static_cast<std::int64_t>(memoryReestimateBytes));
             instrumentation.memoryStatus(E_HardLimit);
             instrumentation.flush();
-            HANDLE_FATAL(<< "Input error: required memory " << bytesToString(memory)
-                         << " exceeds the memory limit " << bytesToString(memoryLimit)
-                         << ". Please force-stop the analysis job, increase the limit to at least "
+            LOG_ERROR(<< "Input error: required memory " << bytesToString(memory) << " exceeds the memory limit "
```

Comment:
Should we wrap the memory values in square brackets? This is something done in ES side. We did not use to do it in c++ but it might be good to start doing so.